reana/reana_dev/git.py

```diff
@@ -815,4 +828,58 @@ def _commit_and_publish_version_bumps(components):
         )
 
 
+@click.option(
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@git_commands.command(name="git-create-pr")
+def git_create_pr_command(component):  # noqa: D301
+    """Creates a GitHub pull request for each selected component.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+    """
+
+    def _git_create_pr(comp):
+        """Create a pull request for the provided component."""
+        for cmd in [
+            "git push origin HEAD",
+            "hub pull-request -p --no-edit",
+            "hub pr list -L 1",
+        ]:
+            run_command(cmd, component)
+
+    for component in select_components(component):
+        if (
+            get_current_branch(get_srcdir(component)) == "master"
+        ):  # replace with is_feature_branch from #371
+            display_message(
+                "You are trying to create PR but the current branch is master, please amend.",
```

Comment:
... please switch to the wanted feature branch.

Also, please add `sys.exit(1)` so that this command would indicate failure.